tests/test_sshfsmount.cpp

```diff
@@ -73,11 +76,20 @@ struct SshfsMount : public mp::test::SftpServerTest
                                      CommandVector::const_iterator& next_expected_cmd, std::string& output,
                                      bool& invoked)
     {
-        auto request_exec = [&commands, &remaining, &next_expected_cmd, &output, &invoked](ssh_channel,
-                                                                                           const char* raw_cmd) {
+        auto request_exec = [this, &commands, &remaining, &next_expected_cmd, &output, &invoked](ssh_channel,
```

Comment:
Why `this`?

Reply:
Because `default_cmds` is a member of `struct SshfsMount`, so we have to pass the object into the lambda.